check-extern.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+EXTERN="$(grep  -e 'extern[^A-Za-z0-9_]' $(find . -name *.c))"
```

Comment:
Maybe sort out comment lines?

Reply:
I could but it might be easier to just remove them